Code/GraphMol/Depictor/DepictUtils.h

```diff
@@ -69,7 +69,7 @@ typedef std::list<PAIR_D_I_I> LIST_PAIR_DII;
   where A is the angle between a and b
  */
 RDKIT_DEPICTOR_EXPORT RDGeom::INT_POINT2D_MAP embedRing(
-    const RDKit::INT_VECT &ring);
+    const RDKit::INT_VECT &ring, const RDKit::INT_VECT &transRingAtoms);
```

Comment:
`embedRing()` is, for better or for worse, in the public API, so we should be thoughtful about changing it's signature.
In this case I don't see the value in calling `findTransRingAtoms()` first and passing in the result; it could just as easily (and efficiently) be called from inside `embedRing()`. This would necessitate moving the utility function `findTransRingAtoms()` to `DepictUtils.cpp`, but that doesn't seem like a big deal.

Reply:
embedRing doesn't know about the mol, so we actually _can't_ check for cis/trans rings inside of it.

I think putting `embedRing()` and `mirrorTransAtoms()` into separate functions would work.